<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSAddBroker</class>
 <widget class="QDialog" name="GBSAddBroker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;</string>
    </property>
    <property name="text">
     <string>添加矩阵设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>2</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../forms/images/gbs/biz/gbs-close-window.png</normaloff>../../forms/images/gbs/biz/gbs-close-window.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>831</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="leActiveNo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>490</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  请输入激活编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRecordCache_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>490</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  请输入激活密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
